statvar_imports/brazil_dados/Auxilio_Brazil_MI_Social/Auxilio_Brazil_Benifits_Program/test_data/sample_output/.csv

```diff
@@ -0,0 +1,141 @@
+observationAbout,observationDate,observationPeriod,value,variableMeasured,unit,measurementQualifier,#input
```

Comment:
Why is the file name ".csv" and ".tmcf", do you need to delete it? I could see under folder "sample_output" folder some are already valid .CSV and .tmcf files.